SpatialGDK/Source/SpatialGDKEditor/Private/EditorExtension/LBStrategyEditorExtension.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EditorExtension/LBStrategyEditorExtension.h"
+#include "LoadBalancing/AbstractLBStrategy.h"
+
+namespace
+{
+	bool InheritFromClosest(UClass* Derived, UClass* PotentialBase, uint32& InOutPreviousDistance)
```

Comment:
Lint - I think we generally don't indent in namespaces and leave a line after and before namespace scope braces.